src/map/skill.cpp

```diff
@@ -16937,7 +16937,6 @@ struct s_skill_condition skill_get_requirement(struct map_session_data* sd, uint
 		case SO_FIRE_INSIGNIA:
 		case SO_WIND_INSIGNIA:
 		case SO_EARTH_INSIGNIA:
-		case WZ_FIREPILLAR: // no gems required at level 1-5 [celest]
 			req.itemid[0] = skill->require.itemid[min(skill_lv-1,MAX_SKILL_ITEM_REQUIRE-1)];
 			req.amount[0] = skill->require.amount[min(skill_lv-1,MAX_SKILL_ITEM_REQUIRE-1)];
 			level_dependent = true;
```

Comment:
If you already add support for this into YAML now:
Shouldn't all of these skills be migrated to the new format?

Also please do not forget to increase the version in the header! And I think you already mentioned yourself that the documentation needs to be changed inside the YAML files.